<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateModelWindow</class>
 <widget class="QDialog" name="CreateModelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Utwórz Model</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>451</width>
     <height>372</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="6" column="2">
     <widget class="QCheckBox" name="checkBox_is_ubm">
      <property name="text">
       <string>model UBM</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="comboBox_distrib_type">
      <item>
       <property name="text">
        <string>Diagonalna macierz kowariancji</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pełna macierz kowariancji</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QTextEdit" name="textEdit_description"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_model_name">
      <property name="text">
       <string>Nazwa modelu: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer_11">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="1">
     <spacer name="verticalSpacer_15">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="2">
     <spacer name="verticalSpacer_16">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer_18">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_distrib_cnt">
      <property name="text">
       <string>Ilość dystrybucji:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_description">
      <property name="text">
       <string>Opis</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <spacer name="verticalSpacer_17">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_distrib_type">
      <property name="text">
       <string>Typ dystrybucji: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer_12">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_distrib_cnt">
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_model_name">
      <property name="maxLength">
       <number>100</number>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancell_button">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_button">
      <property name="text">
       <string>Utwórz</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionCancellButtonClicked">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cancellButtonClicked</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_model_name</tabstop>
  <tabstop>lineEdit_distrib_cnt</tabstop>
  <tabstop>textEdit_description</tabstop>
  <tabstop>comboBox_distrib_type</tabstop>
  <tabstop>checkBox_is_ubm</tabstop>
  <tabstop>cancell_button</tabstop>
  <tabstop>create_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
